src/crypto/shelley/accountingTransactions.js

```diff
@@ -0,0 +1,136 @@
+// @flow
+
+import {InsufficientFunds} from '../errors'
+import {BigNumber} from 'bignumber.js'
+import {
+  OutputPolicy,
+  TransactionBuilder,
+  Address,
+  Input,
+  Value,
+  Fee,
+  TransactionFinalizer,
+  PrivateKey,
+  PublicKey,
+  Witness,
+  SpendingCounter,
+  Hash,
+  Account,
+  AuthenticatedTransaction,
+} from 'react-native-chain-libs'
+
+// TODO: move to external config file
+const CONFIG = {
+  linearFee: {
+    constant: '155381',
+    coefficient: '1',
+    certificate: '4',
+  },
+  addressDiscrimination: {
+    production: '0',
+    test: '1',
+  },
+  genesisHash:
+    'adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770',
+}
+
+// TODO: define a transaction type
+export const buildTransaction = async (
```

Comment:
maybe we could be explicit about being Unsigned so it could be: `buildUnsignedTransaction`